src/main/java/bio/terra/cli/command/workspace/Create.java

```diff
@@ -1,28 +1,30 @@
 package bio.terra.cli.command.workspace;
 
 import bio.terra.cli.auth.AuthenticationManager;
-import bio.terra.cli.context.GlobalContext;
-import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.command.helperclasses.BaseCommand;
+import bio.terra.cli.command.helperclasses.FormatOption;
 import bio.terra.cli.service.WorkspaceManager;
-import java.util.concurrent.Callable;
+import bio.terra.workspace.model.WorkspaceDescription;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the third-level "terra workspace create" command. */
 @Command(name = "create", description = "Create a new workspace.")
-public class Create implements Callable<Integer> {
+public class Create extends BaseCommand {
 
-  @Override
-  public Integer call() {
-    GlobalContext globalContext = GlobalContext.readFromFile();
-    WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
+  @CommandLine.Mixin FormatOption formatOption;
 
-    AuthenticationManager authenticationManager =
-        new AuthenticationManager(globalContext, workspaceContext);
-    authenticationManager.loginTerraUser();
+  /** Create a new workspace. */
+  @Override
+  protected void execute() {
     new WorkspaceManager(globalContext, workspaceContext).createWorkspace();
-    authenticationManager.fetchPetSaCredentials(globalContext.requireCurrentTerraUser());
```

Comment:
Had trouble tracking down the impl of `WorkspaceModel` (seems like it might be generated?), does it serialize cleanly to JSON?

Reply:
Yes, it's a generated class in the WSM client library. It does serialize nicely